docs/Changelog.md

```diff
@@ -29,9 +29,9 @@ sending a mail to <afl-users+subscribe@googlegroups.com>.
      - added neverzero counting to trace-pc/pcgard
      - fixes for laf-intel float splitting (thanks to mark-griffin for
        reporting)
+     - skipping ctors and ifuncs for instrumentation
```

Comment:
for the non-insiders (ctors -> constructors and ifuncs -> LLVM function aliases?)

Reply:
people who dont know ctos wont now constructors either, or?

ifuncs are not function aliases, that is something different.

ifunc is a resolver functionality and program start which allows you to map a function foo to either foo1 or foo2 depending on ... what you decide in your resolver function. so this is code that is run before even constructor(0). just learned about this last week :)